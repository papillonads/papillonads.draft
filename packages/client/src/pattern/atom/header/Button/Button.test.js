import buttonDefault from './Button.render'

describe('<Button />', () => {
  describe('Snaphot', () => {
    test('must match buttonDefault', () => {
      expect(global.renderToJSON(buttonDefault)).toMatchSnapshot()
    })
  })
})
