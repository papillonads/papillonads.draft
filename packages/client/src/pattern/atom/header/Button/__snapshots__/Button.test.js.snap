// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  some children
</div>
`;
